plugins/modules/vmware_appliance_access_info.py

```diff
@@ -103,13 +103,7 @@ def get_mode(module, mode):
 
 def main():
     argument_spec = vmware_httpapi.VmwareRestModule.create_argument_spec()
-    argument_spec.update(
-        access_mode=dict(
-            type="str",
-            choices=["consolecli", "dcui", "shell", "ssh"],
-            default=None,
-        )
-    )
+    argument_spec.update(access_mode=dict(type="str"))
```

Comment:
This change needs to be done out side of this PR.

Reply:
Indeed. I've reversed this change. But I don't plan to land this PR anytime soon. It's more like a backup copy of my work environment.